### PR TITLE
refactor(index): Add templating to main page

### PR DESCRIPTION
**Pull request by Hajar1**
Added bootstrap as a CDN of the home page
resources: templates - decoupled the index from a single HTML file to multiple fragments
resources: stylesheet - Update the stylesheet to more generic CSS syntax

### CODE REVIEW

**Comment by abungay** [comment]
Seems like the search bar & "show x entries" styling is getting a little weird after the introduction of these bootstrap styles.
Before:
Screen Shot 2019-03-12 at 2 03 37 AM
After:
Screen Shot 2019-03-12 at 2 05 11 AM
Also, I noticed that still when we **delete** a book I noticed that the url is not redirecting to "/" as I expected.

**Comment by zaidoon1** [comment]
The url is not redirecting for delete and edit. This is because the fix was only made for creating a book. Also this is an issue with the previous pr that was merged and not this one.

**Comment by abungay** [comment]
> The url is not redirecting for delete and edit. This is because the fix was only made for creating a book. Also this is an issue with the previous pr that was merged and not this one.



> The url is not redirecting for delete and edit. This is because the fix was only made for creating a book. Also this is an issue with the previous pr that was merged and not this one.

Fixed in PR #20.

File: src/main/resources/public/css/stylesheet.css [hunk 1]
```diff
@@ -2,14 +2,14 @@ form  { display: table;      }
 form > p     { display: table-row;  }
 label { display: table-cell; }
 input, textarea { display: table-cell; } 
-.header {
+header {
```
Comment (abungay) [inline src/main/resources/public/css/stylesheet.css:5]: don't know if we want all html header elements to have this styling

File: src/main/resources/public/css/stylesheet.css [hunk 2]
```diff
@@ -2,14 +2,14 @@ form  { display: table;      }
...
     display: flex;
     flex-direction: column;
     align-items: center;
     justify-content: center;
 }
 
-#logo {
+.logo {
```
Comment (abungay) [inline src/main/resources/public/css/stylesheet.css:12]: are we going to have more than one of these logos on a single page?

**Review by LauraMachado21 — DISMISSED** [review]
Navbar already implemented in PR: https://github.com/amazin-team/Amazin-online-bookstore/pull/42

But I like the idea of putting it in a fragment. So, I would wait for the #42 to be merged in and move the navbar fragment out of layout and into a fragment (just like you did).

I also found the branch name misleading. So maybe consider changing your branch name.

File: src/main/resources/public/css/stylesheet.css (hunk 2 above)
Comment (Hajar1) [reply src/main/resources/public/css/stylesheet.css:12]: It is always better to reffer to a class rather then an id as this way we don't pollute the id name field

File: src/main/resources/public/css/stylesheet.css (hunk 1 above)
Comment (Hajar1) [reply src/main/resources/public/css/stylesheet.css:5]: Yes they all will as the header stands for the nav bar

**Review by zaidoon1 — APPROVED** [review]
Nice work!

File: src/main/resources/public/css/stylesheet.css (hunk 1 above)
Comment (abungay) [reply src/main/resources/public/css/stylesheet.css:5]: no actually I was using the classname .header to center the content for the logo, this css selector should not refer to the html 'header' element

**Pull request merged**